packages/vant/src/picker/test/index.spec.tsx

```diff
@@ -440,3 +440,76 @@ test('should render empty slot when options is empty', async () => {
   await wrapper.setProps({ columns: [{ values: ['foo'] }] });
   expect(wrapper.html()).not.toContain('empty content');
 });
+
+test('should reset the second argument when the first argument is changed somehow', async () => {
```

Comment:
A view-based description provides greater clarity. Also, update the PR title accordingly.
```suggestion
test('should reset subsequent columns to their first item when a column changes', async () => {
```